tests/src/Content/Text/HTMLTest.php

```diff
@@ -58,6 +58,10 @@ public function dataHTMLText()
 				'expectedBBCode' => '[audio]http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3[/audio]',
 				'html' => '<audio src="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3" controls="controls"><a href="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3</a></audio>',
 			],
+			'bug-8075-html-tags' => [
+				'expectedBBCode' => "<big rant here> I don't understand tests",
+				'html' => "&lt;big rant here&rt; I don't understand tests",
```

Comment:
```suggestion
				'html' => "&lt;big rant here&gt; I don't understand tests",
```
Not `&rt;`. `lt` stands for `less than`, not `LefT` which would warrant a `rt` as `RighT`.

Reply:
Yeah, I guess I shouldn't watch Youtube videos while coding.